CHANGELOG.md

```diff
@@ -2,8 +2,23 @@
 
 All notable changes to this project will be documented in this file.
 ## [master]
-* Bug: Fix unable to approve application access grant
-* Bug: Fix inconsistent datatype of Members property in Links and replace depricated endpoint
+
+## [2.3.0](https://github.com/Axual/terraform-provider-axual/releases/tag/v2.3.0) - 2024-04-19
```

Comment:
This file in general is duplicating what is already here: https://github.com/Axual/terraform-provider-axual/releases
This changelog file does not add value and I would like to delete it and keep the changelog here because every user will look there: https://github.com/Axual/terraform-provider-axual/releases

Release date is visible from there
Tag date is visible from registry because after every tag push we make a release:
<img width="609" alt="image" src="https://github.com/Axual/terraform-provider-axual/assets/115068027/874cd8b8-5953-4c5e-a40e-d626c2202870">
Tag is also visible from github